### DISCUSSION TITLE
Exit code: 1

### DISCUSSION THREAD

**Discussion by drkknght86** [aclap-dev/video-downloadhelper#371] [Help and Q&A]
### Your question

I keep getting this error 
Exit code: 1

moz-extension://a7ffc732-6fe2-4dff-950b-f2dd5aed0578/background/main.js:1
receive@moz-extension://a7ffc732-6fe2-4dff-950b-f2dd5aed0578/background/main.js:1:13157
nd</callCatchAppNotFound/</<@moz-extension://a7ffc732-6fe2-4dff-950b-f2dd5aed0578/background/main.js:1:42626
Sometimes when i reopen the browser it goes away for one or two video then the error pops back out, ive seen alot of post about this error but haven't found a solid solution yet

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Jason",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by ic37** [comment]
I have exactly this same error message on Firefox 124 running Sonoma MacOS 14.4.1, running latest plugin and coApp. No issues (have been able to download just fine from those websites) until yesterday morning (4/2/2024) when I started getting this error on nearly all downloads. My observations:

- different FireFox versions don't matter (I've tried 123 at least) -- after 2 successful downloads on a cleanly installed browser, I start getting the same error
- different plugin and coApp versions don't matter (I've downgraded to the two previous versions of the plugin before 8.2.2.8 and the previous version of coApp) -- either still fails immediately or will start failing after 2 successful downloads.
- have tried to disable MacOS Firewall. Still same results.

**Reply by ashishdhiman2007** [reply]
I am also facing this exit code 1 issue while downloading `callCatchAppNotFound` on Graphy.com related video hosting

**Reply by Mattmjs** [reply]
@drkknght86  @ic37  @ashishdhiman2007 
Try the beta extension and see if you have the same issue or not.
